<launch>
    <node pkg="arm_pkg" type="camera.py" name="camera" respawn="false" output="screen"/>
    <node pkg="arm_pkg" type="main.py" name="main" respawn="false" output="screen"/>
    <node pkg="arm_pkg" type="unet.py" name="unet" respawn="false" output="screen"/>
    <param name="unet_model_name" type="str" value="unet_model_improveddataset"/>
    <param name="target_epoch" type="int" value="-1"/>
    <param name="image_height" type="int" value="720"/>
    <param name="image_width" type="int" value="1280"/>
</launch>